### PR TITLE
READY: Added NodeJS/JS examples to dev/advanced/bucket-types

### PR DESCRIPTION
**Pull request by lukebakken**

### CODE REVIEW

File: source/languages/en/riak/dev/advanced/bucket-types.md [hunk 1]
```diff
@@ -333,6 +338,18 @@ var rslt1 = client.Get(id1);
 var rslt2 = client.Get(id2);
 ```
 
+```javascript
+client.fetchValue({
+    bucketType: 'type1', bucket: 'my_bucket', key: 'my_key'
+}, function (err, rslt) {
```
Comment (TheOnlyRew) [inline source/languages/en/riak/dev/advanced/bucket-types.md]: Is this the intended indentation level? Looks... well, to say it looks kinda weird doesn't really mean anything when you're talking about JS. It's not what I expected. That might say more about my expectations than the code.

**Comment by TheOnlyRew** [comment]
+1, assuming my question regarding indentation is moot.

**Comment by TheOnlyRew** [comment]
Idle curiosity, does our implementation allow for chained calls? e.g.

```
obj.setContentType('text/plain')
    .setBucketType('no_siblings')
    .setBucket('old_memes')
    .setKey('all_your_base')
    .setValue('all your base are belong to us');
```

File: source/languages/en/riak/dev/advanced/bucket-types.md (hunk 1 above)
Comment (lukebakken) [inline source/languages/en/riak/dev/advanced/bucket-types.md]: I think this is pretty standard actually. Not an expert though.

**Comment by lukebakken** [comment]
http://basho.github.io/riak-nodejs-client/files/lib_commands_kv_riakobject.js.html#l26

**Pull request merged**